packages/genome-nexus-ts-api-client/src/generated/GenomeNexusAPI-docs.json

```diff
@@ -1647,6 +1647,15 @@
         },
         "ncitCode": {
           "type": "string"
+        },
```

Comment:
It's ok to not include `GenomeNexusAPI` related update, we should only include the API updates from your backend pr.

Reply:
Okay!